staff/meetings/minutes

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import os
+import meetings
+from ocflib.misc.shell import get_editor
+from sys import exit
+from datetime import datetime
+from subprocess import call, check_output
+from time import strftime
+
+# note that bod (which is stored in public_html, as they are publicly
+# readable) is symlinked in this folder
+
+# bod template file
+
+
+def main():
+
+    path = os.path.expanduser("~staff/minutes")
+    options = os.listdir(path)
+
+    print("What are you taking notes for?")
+    print("Options:", ", ".join(options))
+
+    choice = input()
+    if choice not in options:
+        return(0)
+
+    path = os.path.join(path, choice)
+
+    if not os.path.isdir(path):
+        os.makedirs(path)
```

Comment:
This is done in the case of a new semester, for example if we have ...../minutes/choice, but not ..../minutes/choice/2017/Spring